### PR TITLE
BigInt+TypedArray tests, splitted

### PR DESCRIPTION
**Pull request by leobalter**
Fixes #1318 

I need to review again the work I've done here. I'm reusing the work from #1318.

### CODE REVIEW

File: test/built-ins/TypedArray/prototype/Symbol.toStringTag/BigInt/invoked-as-func.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 22.2.3.31
```
[the remark this answers is not in the document]
Comment (leobalter) [reply test/built-ins/TypedArray/prototype/Symbol.toStringTag/BigInt/invoked-as-func.js]: I'll do

[the remark this answers is not in the document]
Comment (leobalter) [reply test/built-ins/TypedArray/prototype/Symbol.toStringTag/BigInt/invoked-as-func.js]: this part is already fixed.

**Comment by leobalter** [comment]
cc @rwaldron this should be ready for another review

**Comment by leobalter** [comment]
```
 ./tools/lint/lint.py test/built-ins/TypedArray*
Linting 1859 files
Linting complete. 0 errors found.
```

**Comment by rwaldron** [comment]
Nice work, thanks for wrapping this up!

**Comment by adrianheine** [comment]
`test/built-ins/TypedArray/prototype/set/array-arg-src-tonumber-value-type-conversions.js` is missing the `BigInt` feature flag, see #1455.

**Pull request merged**